rancid-tomatillos/src/Components/Loading/Loading.scss

```diff
@@ -0,0 +1,40 @@
+.loading-wrapper-div {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  height: 100vh;
+  background: {
+    image: url('../../images/Red\ Curtain1.jpg');
+    repeat: no-repeat;
+    size: cover;
+  }
+}
+
+.popcorn-div {
+  width: 25vw;
+  height: 25vw;
+  background: {
+    image: url('../../images/popcorn.png');
+    size: cover;
+    repeat: no-repeat;
+  }
+  animation: pulse 2s infinite;
+}
+
+@keyframes pulse {
```

Comment:
Nice work implementing @keyframes to animate the loading component! Looks really good!